lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs

```diff
@@ -124,9 +117,13 @@ public IEnumerable<Uri> GetChildren(Uri directoryUri)
                 .Select(x => new Uri(x[x.Length - 1]));
         }
 
+
+        /// <summary>
+        /// The Prefix used for URIs on this FileSystem.
+        /// </summary>
         private string GetUriPrefix()
         {
-            return _commandRunner.Run("getconf -confKey fs.default.name").StdOut.First();
+            return _commandRunner.Run("getconf -confKey fs.defaultFS").StdOut.First();         
```

Comment:
Why this change? I left the deprecated one here as it is still supported by Hadoop and it allows us to run on older versions of hadoop.